docs/api.rst

```diff
@@ -8,6 +8,83 @@ Instrument
 .. autoclass:: pysat.Instrument
    :members:
 
+Instrument Methods
+------------------
+
+The following methods support the variety of actions needed
+by underlying pysat.Instrument modules.
+
+Demeter
+^^^^^^^
+
+.. automodule:: pysat.instruments.methods.demeter
+   :members: __doc__, download, bytes_to_float, load_general_header, load_location_parameters, load_attitude_parameters, load_binary_file, set_metadata
+
+
+.. _rst_general_data_general:
+
+General
+^^^^^^^
+
+.. automodule:: pysat.instruments.methods.general
+   :members:
```

Comment:
no members in general?

Reply:
Left those blank so the docs would pull in everything it found.